src/components/MyAccount/RequestsTab/RequestsTab.tsx

```diff
@@ -5,12 +5,16 @@ import type { Hold, Patron } from "../../../types/myAccountTypes"
 import ItemsTab from "../ItemsTab"
 import CancelButton from "./CancelButton"
 import FreezeButton from "./FreezeButton"
+import UpdateLocation from "./UpdateLocation"
+import { filteredPickupLocations as pickupLocations } from "../../../../__test__/fixtures/processedMyAccountData"
```

Comment:
The should probably be in constants.ts instead of fixtures if they're being used outside of tests.

Reply:
Ah yes. That was fixed elsewhere but this was cut from a branch before that. This data actually isn't going to be constant when it's deployed. I'm going to leave it in `src/utils/accountUtils` because thats where it is in main right now.